e2e/grpc-svc-app/grpc/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# We specify the base image we need for our
+# go application
+FROM golang:1.17 AS builder
+# We create an /app directory within our
+# image that will hold our application source
+# files
+RUN mkdir /app
+# We copy everything in the root directory
+# into our /app directory
+ADD . /app
+# We specify that we now wish to execute
+# any further commands inside our /app
+# directory
+WORKDIR /app
+# Avoid the GoProxy
+ENV GOPROXY=direct
+# we run go build to compile the binary
+# executable of our Go program
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64  go build -o grpc ./
+
+# To make our images smaller, use alpine and copy in the service binary.
+FROM alpine:latest
+# Install certs
+RUN apk --no-cache add ca-certificates
+# Copy the binary from the builder image
+COPY --from=builder /app ./
+# Make the binary executable
+RUN chmod +x ./grpc
+# Start the service
+ENTRYPOINT ["./grpc"]
+# The service runs on port 50051
+EXPOSE 50051
```

Comment:
Is this a typical port for grpc?

I wonder if we can do something nice for users and automatically fill the `protocol_version` field in the manifest file to be `grpc` if we detect certain port numbers?

Reply:
I think so.
On the official tutorial, 50051 is usually assigned for server port in any language.
> https://grpc.io/docs/languages/

Reply:
Thanks for the info!